lib/models/Player.js

```diff
@@ -328,6 +353,20 @@ Player.prototype.unMute = function unMute() {
     { mute: 0 });
 };
 
+Player.prototype.unMuteGroup = function unMuteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 0 });
+};
+
+Player.prototype.muteGroup = function muteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 1 });
+};
+
```

Comment:
Seems like I was equally confused when implementing it in the http-api: https://github.com/jishi/node-sonos-http-api/blob/beta/lib/actions/mute.js